constants/map_data_constants.asm

```diff
@@ -9,8 +9,8 @@ MAP_BORDER EQU 3
 	shift_const SOUTH ; 4
 	shift_const NORTH ; 8
 
-; tileset environments
+; flower and water tile animations
```

Comment:
How about: `TILEANIM_NONE`, `TILEANIM_WATER`, `TILEANIM_WATER_FLOWER`

Reply:
Sounds good to me.